checkout/access-checkout-android/src/test/java/com/worldpay/access/checkout/validation/CVVValidatorTest.kt

```diff
@@ -1,289 +1,302 @@
 package com.worldpay.access.checkout.validation
 
-import com.worldpay.access.checkout.api.configuration.CardConfiguration
 import com.worldpay.access.checkout.testutils.CardConfigurationUtil.Brands.VISA_BRAND
 import com.worldpay.access.checkout.testutils.CardConfigurationUtil.Configurations.CARD_CONFIG_BASIC
-import com.worldpay.access.checkout.testutils.CardConfigurationUtil.Defaults.CARD_DEFAULTS
+import com.worldpay.access.checkout.testutils.CardConfigurationUtil.Configurations.CARD_CONFIG_NO_BRAND
 import com.worldpay.access.checkout.testutils.CardNumberUtil.PARTIAL_VISA
 import com.worldpay.access.checkout.testutils.CardNumberUtil.VISA_PAN
 import org.junit.Assert.assertEquals
-import org.junit.Before
 import org.junit.Test
 
-class CVVValidatorImplTest {
+class CVVValidatorTest {
 
-    private lateinit var cvvValidator: CVVValidator
-
-    @Before
-    fun setup() {
-       cvvValidator = CVVValidatorImpl(null)
-    }
-
-    @Test
-    fun `given an empty cvv and null card configuration then should be only completely and partially invalid`() {
-        assertEquals(Pair(ValidationResult(partial = false, complete = false), null), cvvValidator.validate("", null))
-    }
+    private var cvvValidator: CVVValidator = CVVValidator()
 
     @Test
     fun `given a empty cvv and non-null card configuration then should be only partially valid`() {
-        val cvvValidator = CVVValidatorImpl(
-            CardConfiguration(
-                emptyList(),
-                CARD_DEFAULTS
-            )
-        )
-
-        assertEquals(Pair(ValidationResult(partial = true, complete = false), null), cvvValidator.validate("", null))
-    }
-
-    @Test
-    fun `given a non-empty pan, empty cvv and null card configuration then should be completely and partially invalid`() {
-        assertEquals(Pair(ValidationResult(partial = false, complete = false), null), cvvValidator.validate("", "44"))
+        assertEquals(Pair(ValidationResult(partial = true, complete = false), null), cvvValidator.validate(
+            "",
+            null,
+            CARD_CONFIG_NO_BRAND
+        ))
     }
 
     @Test
     fun `given a non-empty cvv, and an empty pan and card configuration then should be completely and partially valid`() {
-        assertEquals(Pair(ValidationResult(partial = true, complete = true), null), cvvValidator.validate("123", null))
+        assertEquals(Pair(ValidationResult(partial = true, complete = true), null), cvvValidator.validate(
+            "123",
+            null,
+            CARD_CONFIG_NO_BRAND
+        ))
     }
 
     @Test
     fun `given a non-empty alphanumeric cvv, and an empty pan and card configuration then should be completely and partially valid`() {
```

Comment:
` then should be completely and partially valid` => name of the test does not match the assertion

I presume that will be reworked as well when we get rid of `partial` so not necessarily worth amending?

Reply:
yeh all of these things will be reworked